layouts/shortcodes/blocks/manifesto.html

```diff
@@ -14,7 +14,10 @@
 	<div class="container" style="max-width: 100vw;">
 		<div class="row">
 			<div class="col mb-5 text-center">
-				<h3>Tekton is a <a href="https://cd.foundation/">Continuous Delivery Foundation</a> project.</h3>
+				<h3>Tekton is a <a href="https://cd.foundation/">Continuous Delivery
+            Foundation</a> project and follows <a
+            href="https://bestpractices.coreinfrastructure.org/en/projects/4020">
```

Comment:
There are individual badges for each project, that badge is only for Tekton Pipeline. So far we also have badges for Chains, CLI, Dashboard, Operator, and Triggers.

Reply:
Changed link to https://github.com/cdfoundation/toc/blob/main/proposals/tekton/graduation.md instead.